src/index.js

```diff
@@ -1,26 +1,34 @@
 // @flow
 import React from "react";
 
+declare function __webpack_require__(pathOrId: string | number): any;
+declare var __webpack_modules__: {};
 type GenericComponent<Props> = Class<React.Component<{}, Props, mixed>>;
 type LoadedComponent<Props> = GenericComponent<Props>;
 type LoadingComponent = GenericComponent<{}>;
 
+let isWebpack = typeof __webpack_require__ !== "undefined";
+let requireFn = isWebpack ? __webpack_require__ : module.require;
+
 let babelInterop = obj => {
   // $FlowIgnore
   return obj && obj.__esModule ? obj.default : obj;
 };
 
 let tryRequire = (pathOrId: string | number) => {
   try {
+    if (isWebpack && !__webpack_modules__[pathOrId]) {
```

Comment:
You only need the `__webpack_modules__[pathOrId]` because you're calling ` tryRequire(serverSideRequirePath);` right? If you don't call that with a `!isWebpack` I don't see why you'd need anything different